website/vercel.json

```diff
@@ -14,6 +14,10 @@
       "source": "/",
       "destination": "/docs"
     },
+    {
+      "source": "/community",
+      "destination": "https://community.cloudquery.io"
+    },
     {
       "source": "/discord",
```

Comment:
Are we keeping these old redirects to Discord?

Reply:
initially I wanted to leave them until we closed the Discord server, but i replaced them with the new one as well.